source/_integrations/todo.markdown

```diff
@@ -61,6 +61,11 @@ Add a new To-do Item. A To-do list `target` is selected with a [Target Selector]
 | Service data attribute | Optional | Description | Example |
 | ---------------------- | -------- | ----------- | --------|
 | `item` | no | the name of the to-do Item. | Submit income tax return
+| `due_date` | yes | The date the to-do item is expected to be completed. | 2024-04-10
+| `due_date_time` | yes | The date and time the to-do item is expected to be completed. | 2024-04-10 23:00:00
```

Comment:
Or just `due`?

Reply:
Sure, but that's more than a name change.

Reply:
The rationale for the current name is only to match the convention of https://www.home-assistant.io/integrations/google/#service-googlecreate_event -- I don't have a preference.

Reply:
To elaborate -- i totally know what you are talking about here, and was thinking about this when adding it.

"due_datetime" would be the way to name this to make it like a python name. 

 "due_date" and "due_date_time" seemed to be reasonable from a user perspective so i went with that name given it was that style in calendar.


(What i'd really love is "due" with a  combined date or date time selector)

Happy to adjust if there are already conventions or preferences.

Reply:
Hmz I see your point, just `due` (combined) won't work because of selectors, so that makes total sense.

I'm personally in the middle if it boils down to `date_time` vs `datetime`, although from a consistency perspective:

<https://www.home-assistant.io/integrations/datetime/>

Reply:
for context: there are currently 11 files in the docs that use `date_time`, and 107 that mention `datetime` (over 400 occurrences).